packages/jsii-pacmak/test/expected.jsii-calc-lib/python/src/scope/jsii_calc_lib/_jsii/__init__.py

```diff
@@ -9,6 +9,7 @@
 import publication
 
 import scope.jsii_calc_base
+import scope.jsii_calc_base_of_base
```

Comment:
How is this change related to out of source builds..?

Reply:
This is due to the new peerDependency